### PR TITLE
Added caniuse hint for queries

### PR DESCRIPTION
**Pull request by mdix**
Readme update as discussed [here](https://github.com/postcss/autoprefixer/issues/657).

### CODE REVIEW

**Comment by ai** [comment]
Making everything bold is very bad 😉

**Comment by ai** [comment]
Also links block should be last in section.

**Comment by mdix** [comment]
Now?

**Comment by ai** [comment]
Why you think that only your section is so important to be hightlighted with bold? 😉 It is very rare mistake (first for few years).

**Comment by mdix** [comment]
If you say it's not, then it's not... 👍

**Comment by ai** [comment]
Now is better. Can you join all commits in one (it is standard ask for all PRs).

**Comment by mdix** [comment]
Ok, should be fine now.

**Comment by ai** [comment]
Awesome! Thanks

File: README.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ You can specify the versions by queries (case insensitive):
 Browserslist works with separated versions of browsers.
 You should avoid queries like `Firefox > 0`.
 
+All queries are based on the [Can I use](caniuse.com) support table, e. g. `last 2 iOS versions` might select `8.4, 9.2, 9.3` (mixed major & minor), whereas `last 2 Chrome versions` might select `50, 49, 48` (major only).
```
Comment (kostasmanionis) [inline README.md]: Umm I find this a bit confusing... Shouldn't `last 2 versions` only match 2 versions instead of 3?

Comment (ai) [inline README.md]: Oops, it is mistake, should be 3. Could you send PR?

Comment (mdix) [inline README.md]: PR: https://github.com/ai/browserslist/pull/57

**Pull request merged**